op-bindings/bindgen/utils.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+)
+
+type contractsData struct {
+	Local     []string           `json:"local"`
+	Etherscan etherscanContracts `json:"etherscan"`
+}
+
+// readJSONFile reads a JSON file from the given `filePath` and unmarshals
+// its content into the provided result interface. It logs the path of the file
+// being read.
+//
+// Parameters:
+// - filePath: The path to the JSON file to be read.
+// - result: A pointer to the structure where the JSON data will be unmarshaled.
+//
+// Returns:
+// - An error if reading the file or unmarshaling fails, nil otherwise.
+func readJSONFile(filePath string, result interface{}) error {
+	log.Printf("Reading contract list from %s\n", filePath)
```

Comment:
We should be mindful of how we log - perhaps its a good idea to use the geth logger and expose the ability to toggle between debug/info etc